### PR TITLE
FEAT: pdu data commands + pbu eng + refactoring of data commands

### LINKED ISSUE #53: FEAT: pdu data commands + pbu eng + refactoring of data commands
As the data commands for the same unit are literally identical apart from the command code, I refactored the functions to one function per unit.

### PR DESCRIPTION
**Pull request by ViciousDoormat**
As the data commands for the same unit are literally identical apart from the command code, I refactored the functions to one function per unit.

### CODE REVIEW

**Comment by ViciousDoormat** [comment]
Warning for the reviewer that GIT made a mess of highlighting the changes

File: src/EPS.cpp [hunk 1]
```diff
@@ -386,10 +391,10 @@ EPS::pdu_overcurrent_reply EPS::get_pdu_overcurrent_fault_state(DWire &wire, uin
     return reply;
 }
 
-EPS::pbu_housekeeping_data_reply EPS::get_pbu_housekeeping_data_raw(DWire &wire, uint8_t i2c_address) {
-    pbu_housekeeping_data_reply reply;
+EPS::pdu_abf_placed_state get_pdu_abf_placed_state(DWire &wire, uint8_t i2c_address) {
```
Comment (ViciousDoormat) [inline src/EPS.cpp:394]: Note, is not a change to the pbu function, but the introduction of a completely new pdu function

File: src/EPS.cpp [hunk 2]
```diff
@@ -427,10 +420,10 @@ EPS::pbu_housekeeping_data_reply EPS::get_pbu_housekeeping_data_raw(DWire &wire,
     return reply;
 }
 
-EPS::pbu_housekeeping_data_reply EPS::get_pbu_housekeeping_data_running_average(DWire &wire, uint8_t i2c_address) {
-    pbu_housekeeping_data_reply reply;
+EPS::pdu_housekeeping_data_reply get_pdu_housekeeping_data(DWire &wire, uint8_t i2c_address, CommandCode commandCode) {
```
Comment (ViciousDoormat) [inline src/EPS.cpp:423]: Note, is not a change to the pbu function, but the introduction of a completely new pdu function

Comment (ViciousDoormat) [reply src/EPS.cpp:423]: be aware of such things

**Comment by DavidGhergut** [comment]
@ViciousDoormat The functionality seems to be correctly implemented! I think you can merge!

**Pull request merged**